torch/_dynamo/output_graph.py

```diff
@@ -334,6 +348,47 @@ def restore_graphstate(self, state: OutputGraphState):
                 removed_nodes += 1
         log.debug("restore_graphstate: removed %s nodes", removed_nodes)
 
+    def add_symbol_bindings(self, arg: GraphArg):
+        # Insert implicit size vars as necessary.  With dynamic shapes, we
+        # maintain the invariant that every sizevar gets a direct SymInt input
+        # into the graph.  This means downstream graph transforms can assume
+        # every size variable is explicitly bound and accessible, instead of
+        # having to pull it out implicitly from tensors.
+
+        if self.export:
+            return
```

Comment:
oh maybe the answer is no, because we don't do any partitioning - so we don't have to worry about "I have a separate bw graph that refers to symbols that don't exist because they were created in the forward graph"

Reply:
If we did want to do dynamic + export + partitioning we would be in some trouble. But we can cross that road when we get there.